extensions/wikia/PaidAssetDrop/js/PaidAssetDrop.desktop.js

```diff
@@ -0,0 +1,138 @@
+define('ext.wikia.PaidAssetDrop', [
```

Comment:
I think the convention is lowerCamelCase, so ext.wikia.paidAssetDrop (and actually that's the extension prefix, so the module should have one more dot and name).